### DISCUSSION TITLE
Welcome to Inter discussions

### DISCUSSION THREAD

**Discussion by rsms** [rsms/inter#424] [Announcements]
## Hi hej hola bonjour 你好 Привет Halló こんにちは xin chào

We’re using GitHub Discussions as a place to connect and have conversations about Inter and related topics.
We hope that you:
- Ask questions you’re wondering about.
- Share ideas.
- [Welcome others and are open-minded](https://github.com/rsms/inter/blob/master/CODE_OF_CONDUCT.md)

**Comment by profujulia** [comment]
Hi Rasmus, 

I work for the Swedish energy and sustainability company Profu (I see that you are Swedish too but I write this in English so that others can read the message in case more people run into the same problem). We love your typeface Inter and have started using it frequently in our presentations. Unfortunately we have run into some problems while doing so. 

In our daily operation we often have to email presentations to costumers and external partners. When we do this, the Inter typeface changes. I have read that it is possible to embed costume typefaces into Powerpoint, Word etc.  See here: https://support.microsoft.com/en-us/office/benefits-of-embedding-custom-fonts-cb3982aa-ea76-4323-b008-86670f222dbc 

To be able to do so, the creator of the typeface must however allow it by setting the edit ability option to "Editable". 
We wonder if it would be at all possible for you to change the settings for Inter to "Editable"? We would really really appreciate it!

Thank you in advance and also thanks for a really lovely typeface!

Kind regards, 
Julia Renstrom from Profu

**Reply by rsms** [reply]
You’re replying to the welcome message.
Please re-post this in Q&A: https://github.com/rsms/inter/discussions/categories/q-a
